src/karma.config.js

```diff
@@ -66,31 +83,13 @@ module.exports = function(config) {
       // Empty HTML file to assist with some tests
       { pattern: './annotator/test/empty.html', watched: false },
 
-      // Karma watching is disabled for these files because they are
-      // bundled with karma-browserify which handles watching itself via
-      // watchify
+      // Test modules.
+      ...testFiles.map(pattern => ({
+        pattern,
 
-      // Unit tests
-      {
-        pattern: 'annotator/**/*-test.coffee',
-        watched: false,
-        included: true,
-        served: true,
-      },
-      {
-        pattern: '**/test/*-test.js',
+        // Disable watching because karma-browserify handles this.
         watched: false,
-        included: true,
-        served: true,
-      },
-
-      // Integration tests
-      {
-        pattern: '**/integration/*-test.js',
-        watched: false,
-        included: true,
-        served: true,
-      },
+      })),
```

Comment:
nb. `included` and `served` are true by default, so we don't need to specify them. See [Karma docs](http://karma-runner.github.io/4.0/config/files.html) for details of what these keys mean.